### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by job13er**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [x] #major# - incompatible API change

The breaking change is the difference in dependencies which will require running the blueprints again. 

# CHANGELOG
### Breaking
* **Swapped**  from `bower` versions of `sinon-chai` and `chai-jquery` to `npm` versions. Running the install blueprint for this addon will remove the bower deps, and add npm deps, as well as clean up `ember-cli-build.js` to undo what it previously added there. If you manually loaded `sinon-chai` and/or `chai-jquery` yourself in `ember-cli-build.js` you'll need to manually undo that. 

### Non-breaking
* **Updated** `eslint-config-frost-standard` and fixed new lint warnings

### CODE REVIEW

**Comment by juwara0** [comment]
👍

File: blueprints/ember-frost-test/index.js [hunk 1]
```diff
@@ -2,16 +2,26 @@
  * Install blueprint for ember-frost-test addon
  */
 
+var Promise = require('bluebird')
```
Comment (sandersky) [inline blueprints/ember-frost-test/index.js]: Why Bluebird over native promises? Doesn't node support those now?

**Comment by sandersky** [comment]
:+1:

File: blueprints/ember-frost-test/index.js (hunk 1 above)
Comment (job13er) [inline blueprints/ember-frost-test/index.js]: I didn't think native promises had a `promisify` or `denodify` helper to convert callback based node methods to promised based methods.

**Pull request merged**